extensions/ql-vscode/src/view/results/alert-table-header.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from "react";
+import { useCallback } from "react";
+import { vscode } from "../vscode-api";
+import {
+  InterpretedResultsSortColumn,
+  InterpretedResultsSortState,
+  SortDirection,
+} from "../../common/interface-types";
+import { nextSortDirection } from "./result-table-utils";
+
+export function AlertTableHeader({
+  sortState,
+}: {
+  sortState?: InterpretedResultsSortState;
+}) {
+  const sortClass = useCallback(
+    (column: InterpretedResultsSortColumn): string => {
```

Comment:
Would it make sense to remove the `column` argument from this function, the `getNextSortState` function and the `toggleSortStateForColumn` function? It seems like it's only ever set to `alert-message`, so removing that argument may simplify this

Reply:
Yes, I agree we can remove the `column` argument. I considered whether we should refactor https://github.com/github/vscode-codeql/blob/6acfb8d15191c7cb9f7f286bb5d13d600139c1c7/extensions/ql-vscode/src/common/interface-types.ts#L277 to a constant so we can reference that here. I came to the conclusion that it's not worth it because these methods in `alert-table-header.tsx` are still referencing `InterpretedResultsSortState` so it will be a type error if we try to return or do a comparison against an incorrect value.